tests/test_glfw_dropfile.c

```diff
@@ -33,10 +33,7 @@ void on_file_drop(GLFWwindow *window, int count, const char **paths) {
     if (!fp) {
         printf("failed to open %s\n", paths[i]);
         perror("fopen");
-#ifdef REPORT_RESULT
-        REPORT_RESULT(0);
-#endif
-        return;
+        assert(false);
```

Comment:
(another case where exiting with a unique id might be nicer for debugging?)

Reply:
Good catch!